source/index.html.md

```diff
@@ -333,64 +333,71 @@ Depending on card scheme and merchant category, the name on the card might be
 necessary for approval of credits. It may be provided through the optional
 parameter `card[name]`.
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without providing
-CSC or other means of cardholder authentication for subsequent payments.
+Clearhaus currently support only one type of subscription billing:
 
-### Subscription concept
+* Recurring: Transactions processed at predetermined, regular intervals not
+    exceeding 1 year; e.g. a monthly subscription for a magazine.
 
-Many PSPs have a subscription concept for supporting recurring payments.
-The first approved authorization is the initial recurring authorization (also
-known as "first in series"), all later authorizations are called subsequent
-authorizations. Clearhaus supports subscriptions in the form of recurring
-payments.
+There may be an agreed end of the series.
 
 ### Repeatedly reserve money
 
-A recurring payment is made by making an authorization and setting `recurring`
-parameter to `true`. The first recurring payment for a given card could be made
-this way (notice that the amount may be zero):
+A first-in-series recurring payment is made by making an authorization and
```

Comment:
```suggestion
A first-in-series recurring payment is initiated by making an authorization and
```

Reply:
It's just that "initiated" is now getting some kind of meaning — "initiating" a transaction is now smelling like "the party that initiated it — CIT or MIT" ... WDYT?

Reply:
I see your point. I only wanted something else than "is made by making" :slightly_smiling_face: 

Perhaps just "is created by making"?

Reply:
```suggestion
A first-in-series recurring payment is created by making an authorization and
```